### PR TITLE
WIP - discussion purposes

### PR DESCRIPTION
**Pull request by mattcollier**
A few fixmes/todos that need specific discussion.  Otherwise, overall comments/suggestions?

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    route,
+    targetApi
+  });
+};
+
+bedrock.events.on('bedrock-express.configure.routes', app => {
+  // TODO: conditionally add rateLimiting API?
+  for(const {
+    authorizationCheck, httpsAgent, method, route, targetApi
+  } of proxies) {
```
Comment (davidlehn) [inline lib/index.js]: Is this taking destructuring too far?  Seems better to make the loop part easier to read:
```js
for(const proxy of proxies) {
   const {authorizationCheck, httpsAgent, method, route, targetApi} = proxy;
   ...
}
```

**Comment by davidlehn** [comment]
Need to make sure the docs note this isn't (currently) dynamic and `addProxy` must be called before `bedrock-express.configure.routes` is emitted.

Does this support streaming?  Doesn't look like it?  Is that going to be a problem?
EDIT: Does passing req.body through like that just magically use streams?

File: lib/index.js (hunk 1 above)
Comment (mattcollier) [reply lib/index.js]: done.

**Comment by dlongley** [comment]
I'm going to merge this into the main branch rather than having it hang out any more. We may not end up releasing or completing this module, but I don't want to lose this work if we change our minds.

**Pull request merged**